tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/game_preferences/multiz_performance.tsx

```diff
@@ -0,0 +1,15 @@
+import { createDropdownInput, Feature } from '../base';
+
+export const multiz_performance: Feature<number> = {
+  name: 'Multiz Rendering Resolution',
+  category: 'GAMEPLAY',
+  description:
+  'How many z levels of multiz to render normally before switching to a less intensive rendering mode',
```

Comment:
Nothing in here means anything to anyone other than people with *very* specific understanding of the code

The description should at the very least prioritize the only question people care about: What do I change for better performance, and what will be different?

Also multiz should be Multi-Z otherwise it looks weird

Reply:
Do you have any suggestions? I thought it was crummy but I wasn't sure how to state it better

Reply:
I have no idea because I'm not one of the people with very specific understanding of the code

Reply:
You would be better off just hiding this behind some arbitrary "detail" slider with instructions to turn detail down with a slow gpu/crummy hardware and up if you have a better GPU

Reply:
you can save the specifics for code comments

Reply:
aight

Reply:
done